cmd/image/qcow2ova/prep/loop.go

```diff
@@ -104,6 +106,26 @@ func mount(opts, src, target string) error {
 func Umount(dir string) error {
 	exitcode, out, err := utils.RunCMD("umount", dir)
 	if exitcode != 0 {
+		if strings.Contains(err, "not mounted") {
+			klog.Infof("Ignoring 'not mounted' error for %s", dir)
+			return nil
+		}
+		if strings.Contains(err, "no mount point specified") {
+			klog.Infof("Ignoring 'no mount point specified' error for %s", dir)
+			return nil
+		}
+		if strings.Contains(err, "target is busy") {
+			for retry := 0; retry < 5; retry++ {
+				exitcode, _, err = utils.RunCMD("umount", dir)
+				if exitcode == 0 || strings.Contains(err, "no mount point specified") || strings.Contains(err, "not mounted") {
+					return nil
+				}
+			}
+			exitcode, out, err = utils.RunCMD("umount", "-lf", dir)
```

Comment:
Lets add some info message before lazy mount to inform user that we are going ahead and doing lazy unmount because of so and so reason..

Reply:
Done.